README.md

```diff
@@ -34,6 +34,9 @@ Below are the mutation operators available in the current version of WAF-A-MoLE.
 | Integer Encoding | `admin' OR 1=1#` ⇒ `admin' OR 0x1=(SELECT 1)#`|
 | Operator Swapping | `admin' OR 1=1#` ⇒ `admin' OR 1 LIKE 1#`|
 | Logical Invariant | `admin' OR 1=1#` ⇒ `admin' OR 1=1 AND 0<1#`|
+| Number Shuffling | `admin' OR 1=1#` ⇒ `admin' OR 2=1#`|
+| Base Shuffling | `admin' OR 1=1#` ⇒ `admin' OR 0x8b=1#`|
+| Symbol Injection | `admin' OR 1=1#` ⇒ `admin'/OR}1=1#`|
```

Comment:
Were you referring to the Symbol Injection operator in this comment? Might be better to drop that one if so

Reply:
Yep, I think Symbol Injection does not preserve semantics, and in general, I am afraid that it might create unwanted syntax errors (I think the one in the example is a syntax error itself).
It would work if we add some syntax checks before confirming the mutation, but I am not sure it would be worth it at this point, at least performance-wise.

I think dropping it for the moment and reinserting it at a later time would be the best choice.
@zangobot do you agree?

Reply:
I agree, it does sound pretty similar. Maybe another scenario could be one similar to the string version of Number Shuffling?

In this mutation, symbols could be added symmetrically in both sides of an equation, e.g.:
`'aaa'='aaa'` => `'a/aa'='a/aa'`

But I guess it is a little far fetched, I'm not completely sure this might be useful as it is, maybe it could be a separate mutation strategy.